router/index.js

```diff
@@ -24,20 +31,102 @@ router.post('/login', (req, res, next) => {
 
 // get list of cameras
 
-router.get('/camera', (req, res, next) => {
-  res.sendStatus(200);
+router.get('/camera', async (req, res, next) => {
+
+  try {
+    const allcameras = await db.getAllCameras();
+
+    if (allcameras.length === 0) {
+      res.json({message: 'no cameras available'});
+    } else {
+      res.send(allcameras);
```

Comment:
I think you should use a res.json here

Reply:
The difference is pretty minor though. More info here: https://stackoverflow.com/questions/19041837/difference-between-res-send-and-res-json-in-express-js

Reply:
thanks for the feedback. I just forgot to change it to res.json even though it still worked as it is already a json object using res.send